src/bun.js/bindings/bindings.cpp

```diff
@@ -3640,7 +3640,14 @@ JSC__JSValue JSC__JSValue__getIfPropertyExistsFromPath(JSC__JSValue JSValue0, JS
                 jc = pathString.characterAt(j);
             }
 
-            PropertyName propName = PropertyName(Identifier::fromString(vm, pathString.substring(i, j - i)));
+            StringView propNameStr = pathString.substring(i, j - i);
```

Comment:
I don't think this is a good fix

This should be: 
```suggestion
            String propNameStr = pathString.substring(i, j - i);
            PropertyName propName = Identifier::fromString(vm, propNameStr);
```

Reply:
The reason the below works is because Identifier::fromString(vm, const LChar*, unsigned) creates a new AtomString

Reply:
Updated to use `String`